DockerfileLightCaddy

```diff
@@ -0,0 +1,4 @@
+FROM caddy:alpine
```

Comment:
See above comment in Caddyfile, if you can provide a good reason as to swapping over to Caddy I'm all ears, but considering this application only uses a webserver to receive (this is not the reverse proxy, that is done by actions outside this repo) to G5V, and I do believe that NGINX is performant enough to handle this task, and is still more widely accepted at this time. 

SSL is not needed within this container, since it will be operating internally, and the reverse proxy/load balancer/anything operating in front of this app should be taking care of the SSL requests and just handing it off to this app. So to me, there is no added benefit of swapping over from NGINX to Caddy in this particular instance. So if you wouldn't mind, please remove for now and if needed it can be added in a future pull request :)

Thanks!

Reply:
This Dockerfile is not used by default. Just as an option. That is why is has caddy in the name. If you want, I can cherry-pick that file to another pull request, as you suggested, but I do not see a reason, why that could not be an option.

Reply:
Cherry picking it would be preferred, as I would like to keep different features separated in different PRs :)

Reply:
Removed from here :)